PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+
+using Microsoft.Office.Interop.PowerPoint;
+
+using PowerPointLabs.Models;
+
+namespace PowerPointLabs.SyncLab
+{
+    public class SyncLabShapeStorage : PowerPointPresentation
+    {
+#pragma warning disable 0618
+
+        private int nextKey = 0;
+
+        public SyncLabShapeStorage() : base()
+        {
+            Path = Globals.ThisAddIn.PrepareTempFolder(
```

Comment:
I changed it to `System.IO.Path.GetTempPath()`, similar to what Util/Graphics does, because I realised we don't need the files to be in their own folder.